near-sdk/compilation_tests/schema_derive_invalids.stderr

```diff
@@ -58,14 +46,11 @@ error: `NearSchema` does not support derive for unions
 43 | | }
    | |_^
 
-error: `NearSchema` does not support derive for unions
-  --> compilation_tests/schema_derive_invalids.rs:47:1
+error: Unknown field: `schemars`
+  --> compilation_tests/schema_derive_invalids.rs:46:13
    |
-47 | / union UnsupporteedWithArgs {
-48 | |     a: u8,
-49 | |     b: u16,
-50 | | }
-   | |_^
```

Comment:
In order to retain the test to check what it used to check, change the `#[abi(json, schemars)]` to just `#[abi(json)]`: https://github.com/near/near-sdk-rs/blob/9e79cea7b87f56a76f9f99f2d86fe646e631a4fc/near-sdk/compilation_tests/schema_derive_invalids.rs#L46